waspc/src/Wasp/Generator/Job.hs

```diff
@@ -28,4 +28,4 @@ data JobMessageData
 
 data JobOutputType = Stdout | Stderr deriving (Show, Eq)
 
-data JobType = WebApp | Server | Db deriving (Show, Eq, Ord, Bounded, Enum)
+data JobType = WebApp | Server | Db | Cli deriving (Show, Eq, Ord, Bounded, Enum)
```

Comment:
Woah woah new job type here, in Generator :D. These job types are normally to indicate the context in which Job is running. I am not sure how much sense it makes to introduce CLI, what is that, what does it stand for?